src/dso_api/dynamic_api/serializers/base.py

```diff
@@ -259,6 +257,12 @@ def _apply_permission(self, field_name: str, field: serializers.Field) -> bool:
             # Check access
             permission = user_scopes.has_field_access(field_schema)
             if not permission:
+                logging.debug(
```

Comment:
```suggestion
                logging.info(
```

This is security-related. In fact, I suspect we should audit-log it, but we can leave that out of this PR.